Make extension configurable via XML (#51)
hivemq/hivemq-mqtt-message-log-extension: README.adoc

* Download the extension from the {hivemq-extension-download}[HiveMQ Marketplace^].
* Copy the content of the zip file to the `extensions` folder of your HiveMQ nodes.

> * Add a configuration file named `config.xml` file to the extension `conf` folder.

Reviewer: ```suggestion
* Configure the extension by adding a `config.xml` file in the extensions `conf` folder.
```

** For your convenience, we provide an example configuration `conf/examples/config.xml` that you can copy and modify as desired.
** The file must be located in `HIVEMQ_HOME/extensions/hivemq-mqtt-message-log-extension/conf/config.xml`.

> NOTE: Release 1.2.0 deprecates the legacy configuration file `mqttMessageLog.properties`.

Reviewer: ```suggestion
NOTE: Starting with release 1.2.0 of the extension, the legacy configuration file `mqttMessageLog.properties` is deprecated. Support for the legacy configuration will be remove in a future release.
```

> We encourage you to use `conf/config.xml` instead, as we will remove support for the legacy configuration in a future release.

Reviewer: ```suggestion
```

== Configuration

=== Event Configuration

By default, all MQTT events are logged.
It is possible to opt out of specific log event types by adding a `config.xml` file to your `hivemq-mqtt-message-log-extension/conf` folder.

> There is an example file available in `conf/examples/` which removes the logging of PUBLISH messages.

Reviewer: ```suggestion
An example configuration file is available in `conf/examples/`, which removes the logging of PUBLISH messages.
```
